cmd/status-cli/main.go

```diff
@@ -0,0 +1,347 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/google/uuid"
+
+	"github.com/status-im/status-go/account/generator"
+	"github.com/status-im/status-go/appdatabase"
+	"github.com/status-im/status-go/common/dbsetup"
+	gethbridge "github.com/status-im/status-go/eth-node/bridge/geth"
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/multiaccounts"
+	"github.com/status-im/status-go/protocol"
+	"github.com/status-im/status-go/protocol/common"
+	"github.com/status-im/status-go/protocol/protobuf"
+	"github.com/status-im/status-go/protocol/requests"
+	"github.com/status-im/status-go/t/helpers"
+	"github.com/status-im/status-go/wakuv2"
+	"github.com/status-im/status-go/walletdatabase"
+
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := &cli.App{
+		Commands: []*cli.Command{
+			{
+				Name:    "dm",
+				Aliases: []string{"d"},
+				Usage:   "Send direct message",
+				Action: func(cCtx *cli.Context) error {
+					fmt.Println("params: ", cCtx.Args().First())
+
+					// TODO test sharding
+					// enrBootstrap := "enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.test.shards.nodes.status.im"
+					enrBootstrap := "enrtree://AL65EKLJAUXKKPG43HVTML5EFFWEZ7L4LOKTLZCLJASG4DSESQZEC@prod.status.nodes.status.im"
+
+					// Start alice node and messager
+
+					fmt.Println("[Alice] starting messager")
+					alicePrivKey, err := crypto.GenerateKey()
+					if err != nil {
+						fmt.Println(err)
+						return err
+					}
+
+					config := &wakuv2.Config{}
+					config.EnableDiscV5 = true
+					config.DiscV5BootstrapNodes = []string{enrBootstrap}
+					config.DiscoveryLimit = 20
+					aliceNode, err := wakuv2.New("", "", config, nil, nil, nil, nil, nil)
```

Comment:
This and `bobConfig` (lines 139/143) can probably be extracted to a function to avoid duplication